src/index.css

```diff
@@ -152,9 +152,16 @@ hr {
 }
 
 .flow-images {
-  max-width: 55%;
```

Comment:
@liu-caro do we need to keep this and the other one below that's deleted?

Reply:
If it's needed and we can keep it, those will be the standard dimensions for all .flow-images; I refactored the code so that each page's custom css sets specific width and height properties, where .flow-images in index.css only takes care of basic styling and the mobile view that also prevents that bug from popping up

Reply:
This seems fine to me! Thanks for bringing it up @feliciazhang